query/codebuild/codebuild_project_with_user_controlled_buildspec.sql

```diff
@@ -0,0 +1,16 @@
+select
+  -- Required Columns
+  arn as resource,
+  case
+    when split_part(source ->> 'Buildspec', '.', -1) = 'yaml' then 'ok'
```

Comment:
Should we only check for `yaml` or we can add check for `yml` also?

Reply:
yes only for `yaml`